docs/cimpl6Authoring.md

```diff
@@ -39,56 +37,56 @@ Model key:
 * Dark green: models created using CIMPL, leveraging OBF
```

Comment:
I don't like this figure. It has too many specifics (mCODE, Skin and Wound) that will be confusing. Better to say "IG 1" "IG 2" without being specific. See the similar figure in the Tooling Reference. (Data-model-use-case.png)

Reply:
OK, will get the original from May and propose a new version.

Reply:
@markkramerus   I reviewed the diagram in the tooling reference - suggest replacing the current one in the user guide with the one from the tooling reference.  Assume your reference is to the one "Common Models Promote Interoperability Across Implementation Guides"?
Common Models Promote Re-Use and Interoperability Across Implementation Guides?

Reply:
Yes, either one.

Reply:
Done